components/cards/cards.html

```diff
@@ -0,0 +1,275 @@
+							style="
+								height: 230px;
+								border: 0;
+								transform: scale(1);
+								overflow: hidden;
+								margin-top: 2rem;
+								margin-bottom: 4rem;
+							"
+							sandbox="allow-scripts allow-same-origin"
+						>
+						</iframe>
+
+						<!-- card with text overlay -->
+						<div class="intro-heading lg-text">Card with text overlay</div>
+						<p class="intro-sub-heading sm-text">
+							To overlay text over other items, you need to add text-overlay
+							class to the text which you want to overlay.
+						</p>
+						<div class="card card-overlay card-shadow">
+							<img
+								src="../../assets/illustration-hero.svg"
+								alt="card-example"
+							/>
+							<h1 class="text-overlay">Card Overlay</h1>
+							<p class="sm-text">
+								Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+								Voluptates ut pariatur vel quod maxime necessitatibus!
+							</p>
+						</div>
+						<iframe
+							src="https://carbon.now.sh/embed?bg=rgba%28171%2C184%2C195%2C1%29&t=seti&wt=none&l=auto&width=774&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=18px&ph=44px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card%2520card-overlay%2522%253E%250A%2509%253Cimg%2520src%253D%2522your-img%2522%2520alt%253D%2522card-example%2522%252F%253E%250A%2509%253Ch1%2520class%253D%2522text-overlay%2522%253ECard%2520Overlay%253C%252Fh1%253E%250A%2509%253Cp%253E%2520text%2520here%2520%253C%252Fp%253E%250A%253C%252Fdiv%253E"
+							style="
+								height: 250px;
+								border: 0;
+								transform: scale(1);
+								overflow: hidden;
+								margin-top: 2rem;
+								margin-bottom: 4rem;
+							"
+							sandbox="allow-scripts allow-same-origin"
+						>
+						</iframe>
+
+						<!-- horizontal card -->
+						<div class="intro-heading lg-text">Horizontal Card</div>
+						<p class="intro-sub-heading sm-text">
+							To create a horizontal card add class card-horizontal to the card.
+						</p>
+						<div class="card-horizontal card-shadow">
+							<img
+								src="../../assets/illustration-hero.svg"
+								alt="card-example"
+							/>
+							<div>
+								<h1 class="md-text">Horizontal card</h1>
+								<p class="sm-text">
+									Lorem ipsum dolor sit amet consectetur adipisicing elit.
+									Velit, laudantium.
+								</p>
+							</div>
+						</div>
```

Comment:
The image looks squeezed here. You can fix that by giving the image this property `object-fit: cover;` in the css.

![Screenshot 2022-02-02 at 18 19 51](https://user-images.githubusercontent.com/62737267/152157519-541a69ad-ef00-4751-b855-d7c18a411c51.png)

Reply:
i have given object-fit to the img.